pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -5468,3 +5468,32 @@ SELECT ST_PointInsideCircle(ST_Point(1,2), 0.5, 2, -3);
 
 statement error pq: st_pointinsidecircle\(\): first parameter has to be of type Point
 SELECT ST_PointInsideCircle(ST_GeomFromText('LINESTRING (-0.5 0.5, 0.5 0.5)'), 0.5, 2, -3);
+
+
+subtest st_linelocatepoint
+
+query R
+SELECT ROUND(ST_LineLocatePoint(line, point)::float, 2) FROM ( VALUES
```

Comment:
the majority of these should be a unit test in the pkg/geo/geomfn package. We only need one or two tests here to prove it works end to end.

Reply:
Added more unit tests to get new code covered.